config/answers.sam-installer.yaml

```diff
@@ -3,7 +3,9 @@ certs:
   generate: true
   deploy: true
   group: foreman
-katello: true
+katello:
+  package_names:
+    - katello-sam
```

Comment:
Think of it less like dependencies, and more like just tracking and ensuring packages are installed to ensure a particular state of the application. We *don't* have to include both. We just do for Katello.